test/functional/wallet_importdescriptors.py

```diff
@@ -120,12 +119,11 @@ def run_test(self):
 
         self.log.info("Internal addresses should be detected as such")
         key = get_generate_key()
-        addr = key_to_p2pkh(key.pubkey)
         self.test_importdesc({"desc": descsum_create("pkh(" + key.pubkey + ")"),
                               "timestamp": "now",
                               "internal": True},
                              success=True)
-        info = w1.getaddressinfo(addr)
+        info = w1.getaddressinfo(key.p2pkh_addr)
         assert_equal(info["ismine"], True)
         assert_equal(info["ischange"], True)
```

Comment:
as you are touching this; for some reason, this is the only one of the related subtests that the `test_address()` helper method is not used.